bedrock/products/templates/products/vpn/platforms/ios.html

```diff
@@ -0,0 +1,167 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "products/vpn/includes/macros.html" import vpn_nav_platform_cta, vpn_platform_content_well with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% set vpn_countries = 30 %}
+{% set vpn_devices = 5 %}
+{% set vpn_servers = 280 %}
+
+{% block page_title_full %}{{ ftl('vpn-ios-page-title') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block page_desc %}{{ ftl('vpn-ios-page-description') }}{% endblock %}
+
+{% block body_id %}mozilla-vpn-platforms-ios{% endblock %}
+
+{% set _utm_source = 'www.mozilla.org-vpn-ios-page' %}
+{% set _utm_campaign = 'vpn-product-page' %}
```

Comment:
Should utm campaign be something specific for these pages?

Reply:
The `_utm_source` above is.

Reply:
@rraue Do you think I should change all of these to `vpn-platform-page`?